src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -394,11 +368,13 @@ class SchedulerActions(
     (instances.allSpecIdsWithInstances -- runSpecIds).foreach { unknownId =>
       logger.warn(
         s"RunSpec $unknownId exists in InstanceTracker, but not store. " +
-          "The run spec was likely terminated. Will now expunge."
+          "The run spec was likely terminated. Will now decommission."
```

Comment:
this log warning is a bit confusing (which I'm only noticing because no good deed goes unpunished).

I think maybe this would be a little clearer?

"An instance references runSpec $unknownId, which does not exist. Will now decommission."